examples/kubernetes-vmss/kubernetes.json

```diff
@@ -11,7 +11,8 @@
     "masterProfile": {
       "count": 1,
       "dnsPrefix": "",
-      "vmSize": "Standard_D2_v2"
```

Comment:
Could we have a default example which is of the current way - master with no vmss and agents with vmss. and then have one which is kubernetes-master-vmss.json which has this. Are we expecting this to become default configuration for customers from what we are seeing in the field ?